tools/functional-tester/etcd-tester/election_runner_stresser.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"sync"
+
+	"github.com/coreos/etcd/tools/functional-tester/etcd-runner/runner"
+)
+
+type electionRunnerStresser struct {
+	cf     *runner.EtcdRunnerConfig
+	runWg  sync.WaitGroup
+	errch  chan error
+	ctx    context.Context
+	cancel func()
+}
+
+func (ers *electionRunnerStresser) Stress() error {
+	ers.ctx, ers.cancel = context.WithTimeout(context.Background(), runnerTimeout)
+	ers.errch = make(chan error, 2)
```

Comment:
1
https://github.com/fanminshi/etcd/blob/e84ab64c17f22fc8986cd919223a27f036023a39/tools/functional-tester/etcd-tester/election_runner_stresser.go#L39-L39

2
https://github.com/fanminshi/etcd/blob/e84ab64c17f22fc8986cd919223a27f036023a39/tools/functional-tester/etcd-tester/election_runner_stresser.go#L49

Now I see that error checking at [2] might not be needed since the the same error will be recorded at [1]